js_course/test-1.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+
+test('test-1', async ({ page }) => {
```

Comment:
Could you please separate your tests by block following AAA (https://thinkster.io/tutorials/blogs/the-aaa-structure-of-tests)
For example,

```
await page.goto('https://www.saucedemo.com/');
// empty line
await page.locator('[data-test="username"]').click();
// empty line
await expect(page.locator('.title')).toHaveText('Products');
```